jest.web.config.js

```diff
@@ -30,9 +27,24 @@ module.exports = {
     ...base.testPathIgnorePatterns,
     '.*(/tests?/.*.(native|ios|android).(test|spec)).(tsx?)$',
   ],
-  transform: {
-    ...base.transform,
-  },
+  transform: { ...base.transform },
+  transformIgnorePatterns: [
+    'node_modules/(?!' +
+      'firebase' +
+      '|@firebase' +
+      '|@ptomasroos/react-native-multi-slider' +
+      '|react-native-svg-web' +
+      '|react-native-animatable' +
+      '|react-native-web' +
+      '|react-native-modal' +
+      '|react-native-calendars' +
+      '|react-native-swipe-gestures' +
+      '|react-native-permissions' +
+      '|react-native-qrcode-svg' +
+      '|react-native-country-picker-moda' +
+      '|instantsearch.js' +
+      ')',
+  ],
```

Comment:
Je serai d'avis de ne pas écraser la valeur qui vient de `jest.config.js`, mais si on peut pas faire sans c'est pas très grave.

Sinon, petite correction :
```suggestion
  transformIgnorePatterns: [
    'node_modules/(?!' +
      'firebase' +
      '|@firebase' +
      '|@ptomasroos/react-native-multi-slider' +
      '|react-native-svg-web' +
      '|react-native-animatable' +
      '|react-native-web' +
      '|react-native-modal' +
      '|react-native-calendars' +
      '|react-native-swipe-gestures' +
      '|react-native-permissions' +
      '|react-native-qrcode-svg' +
      '|react-native-country-picker-modal' +
      '|instantsearch.js' +
      ')',
  ],
```